corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/entities/app_selection.js

```diff
@@ -0,0 +1,62 @@
+/*global FormplayerFrontend */
+
+FormplayerFrontend.module("Entities", function (Entities, FormplayerFrontend, Backbone, Marionette, $) {
+    Entities.AppSelect = Backbone.Model.extend({
+        urlRoot: "appSelects",
+    });
+
+    Entities.configureStorage("FormplayerFrontend.Entities.AppSelect");
+
+    Entities.AppSelectCollection = Backbone.Collection.extend({
+        url: "appSelects",
+        model: Entities.AppSelect,
+    });
+
+    Entities.configureStorage("FormplayerFrontend.Entities.AppSelectCollection");
+
+    var storeApps = function (apps) {
+        var old_apps = new Entities.AppSelectCollection();
+        var defer = $.Deferred();
+        old_apps.fetch({
+            success: function (data) {
+                defer.resolve(data);
+            },
+        });
+        var promise = defer.promise();
+        $.when(promise).done(function (oldApps) {
```

Comment:
if you just made this the success function you can do away with the `promise` abstraction`